mission_control/navigator_missions/nav_missions/find_the_break.py

```diff
@@ -1,11 +1,156 @@
 #!/usr/bin/env python
+from __future__ import division
 import txros
+from twisted.internet import defer
+from sensor_msgs.msg import Image
 import numpy as np
-from navigator_tools import fprint
+from navigator_tools import fprint, Debug
+from cv_bridge import CvBridge, CvBridgeError
+from collections import Counter
+import numpy.ma as ma
+
+import cv2
+
+
+class FindTheBreakMission():
+
+    def __init__(self, nh):
+        self.nh = nh
+        self.image = None
+        self.debug = Debug(nh=nh)
+        self.bridge = CvBridge()
+
+    @txros.util.cancellableInlineCallbacks
+    def init_(self):
+        yield self.nh.subscribe("/video_player/croppedFile/image_raw", Image, self.image_cb)
+        defer.returnValue(self)
+
+    def image_cb(self, image):
+        self.image = self.bridge.imgmsg_to_cv2(image, "bgr8")
+
+    @txros.util.cancellableInlineCallbacks
+    def get_direction(self):
+        cap = cv2.VideoCapture("/home/tess/bags/croppedFile.mp4")
```

Comment:
This seems a little un-rosifed and disconnected from the mission. It should subscribe to an image topic. Maybe you want this perception to be a different class/node?